common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1566,33 +1574,70 @@ object RssConf extends Logging {
     }
   }
 
-  def checkSlotsFinishedInterval(conf: RssConf): Long = {
-    conf.getTimeAsMs("rss.worker.checkSlots.interval", "1s")
-  }
+  val WORKER_GRACEFUL_SHUTDOWN_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.worker.graceful.shutdown.enabled")
+      .withAlternative("rss.worker.graceful.shutdown")
+      .categories("worker")
+      .doc("When true, during worker shutdown, the worker will wait for all released slots " +
+        s"to be committed or destroyed.")
+      .version("0.2.0")
+      .booleanConf
+      .createWithDefault(false)
 
-  def checkSlotsFinishedTimeoutMs(conf: RssConf): Long = {
-    conf.getTimeAsMs("rss.worker.checkSlots.timeout", "480s")
-  }
+  val WORKER_GRACEFUL_SHUTDOWN_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.graceful.shutdown.timeout")
+      .withAlternative("rss.worker.shutdown.timeout")
+      .categories("worker")
+      .doc("The worker's graceful shutdown timeout time.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("600s")
 
-  def workerGracefulShutdown(conf: RssConf): Boolean = {
-    conf.getBoolean("rss.worker.graceful.shutdown", false)
-  }
+  val WORKER_CHECK_SLOTS_FINISHED_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.worker.graceful.shutdown.checkSlotsFinished.interval")
+      .withAlternative("rss.worker.checkSlots.interval")
+      .categories("worker")
+      .doc("The wait interval of checking whether all released slots to be committed or destroyed during worker graceful shutdown")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("1s")
 
-  def shutdownTimeoutMs(conf: RssConf): Long = {
-    conf.getTimeAsMs("rss.worker.shutdown.timeout", "600s")
-  }
+  val WORKER_CHECK_SLOTS_FINISHED_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.graceful.shutdown.checkSlotsFinished.timeout")
+      .withAlternative("rss.worker.checkSlots.timeout")
+      .categories("worker")
+      .doc("The wait time of waiting for the released slots to be committed or destroyed during worker graceful shutdown.")
+      .version("0.2.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("480s")
 
-  def workerRecoverPath(conf: RssConf): String = {
-    conf.get("rss.worker.recoverPath", s"${System.getProperty("java.io.tmpdir")}/recover")
-  }
+  val WORKER_RECOVER_PATH: ConfigEntry[String] =
+    buildConf("celeborn.worker.graceful.shutdown.recoverPath")
+      .withAlternative("rss.worker.recoverPath")
+      .categories("worker")
+      .doc("The path to store levelDB.")
+      .version("0.2.0")
+      .stringConf
+      .transform(_.replace("/tmp", System.getProperty("java.io.tmpdir")))
```

Comment:
if the user-provided path is "/mnt/shuffle/tmp", it will cause unexpected behavior. please use `<tmp>` as the placeholder, and document the resolve logic the "doc('xxx')"

Reply:
Done